hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <iostream>
+#include "DomoticzHardware.h"
+
+class CThermostatHardware : public CDomoticzHardwareBase
+{
+public:
+
+	//thermostat available mode in string "OFF,ECO,CONFOR,AUTO," separated by comma, 
```

Comment:
Use English instead of French abbreviation?
```suggestion
	//thermostat available mode in string "OFF,ECO,COMFORT,AUTO," separated by comma, 
```